packages/resource/README.md

```diff
@@ -11,10 +11,10 @@
 
 A collection of composable primitives to augment [`createResource`](https://www.solidjs.com/docs/latest/api#createresource)
 
-- [`makeAbortable`](#makeabortable) - wraps the fetcher to be abortable and auto-abort on re-fetch or timeout
-- [`makeAggregated`](#makeaggregated) - wraps the resource to aggregate data instead of overwriting it
-- [`makeCache`](#makecache) - wraps the fetcher to cache the responses for a certain amount of time
-- [`makeRetrying`](#makeretrying) - wraps the fetcher to retry requests after a delay
+[`createAggregated`](#createaggregated) - wraps the resource to aggregate data instead of overwriting it
+[`makeAbortable`](#makeabortable) - wraps the fetcher to be abortable and auto-abort on re-fetch or timeout
+[`makeCache`](#makecache) - wraps the fetcher to cache the responses for a certain amount of time
+[`makeRetrying`](#makeretrying) - wraps the fetcher to retry requests after a delay
```

Comment:
if you don't put `-` in front, or add extra empty line between they get rendered like this:

![image](https://user-images.githubusercontent.com/24491503/228473238-610aefe2-286b-4561-b312-d6215ff45134.png)